docs/book/src/topics/troubleshooting.md

```diff
@@ -59,3 +59,144 @@ $ aws iam get-instance-profile --instance-profile-name control-plane.cluster-api
 
 ```
 If instance profile does not look as expected, you may try recreating the CloudFormation stack using `clusterawsadm` as explained in the above sections.
+
+
+## Recover a management cluster after losing the api server load balancer
+
+### **Access the api server locally**
+
+1. ssh to a control plane node and modify the `/etc/kubernetes/admin.conf`
+
+    * Replace the `server` with `server: https://localhost:6443`
+
+    * Add `insecure-skip-tls-verify: true`
+
+    * Comment out `certificate-authority-data:`
+
+2. Export the kubeconfig and ensure you can connect
+
+    ```bash
+    export KUBECONFIG=/etc/kubernetes/admin.conf
+    kubectl get nodes
+    ```
+
+
+### **Get rid of the lingering duplicate cluster**
```

Comment:
This is not explicit, would be good to explain how you end up in this situation right after `## Recover a management cluster after losing the api server load balancer` title.

Also, there is no need to have a lingering Cluster, this workaround works for accidentally manually deleting the api-server LB.

Reply:
Thanks, I made some updates. I think it's important to keep the lingering cluster section but I put a statement at the start to explain that it's only needed in a certain scenario